soundfile.py

```diff
@@ -167,6 +167,33 @@
     __libsndfile_version__ = __libsndfile_version__[len('libsndfile-'):]
 
 
+class SoundFileError(RuntimeError):
+    """Base class for all SoundFile-specific errors.
```

Comment:
Again, a matter of taste: "SoundFile" is the main class `soundfile.SoundFile`, "soundfile" is the name of the module.

Here, we are talking about the module, right?

Reply:
Right.